src/views/ContactView.jsx

```diff
@@ -1,21 +1,48 @@
 import React from "react";
 import { Helmet } from "react-helmet-async";
 import { useSelector } from "react-redux";
-
+import { useMediaQuery, useTheme, Grid } from "@material-ui/core";
 import ScreenSplit from "../components/ScreenSplit";
 import ContactUs from "../components/ContactUs";
+import ContactForm from "../components/ContactForm";
 
 const ContactView = () => {
+  const theme = useTheme();
   const { contact } = useSelector((contact) => contact.appData);
-
+  const mobile = useMediaQuery(theme.breakpoints.down("md"));
+  
   return (
     <>
       <Helmet>
         <title>Contact us</title>
       </Helmet>
-      <ScreenSplit left={<ContactUs data={contact} />} />
+      {mobile ? (
+        <Grid container style={styles.alignVertically}>
+          <Grid item>
+            <ContactUs data={contact} />
+          </Grid>
+          <Grid item>
+            <ContactForm />
+          </Grid>
+        </Grid>
+      ) : (
+        <ScreenSplit
+        left={<ContactUs data={contact} />}
+        right={<ContactForm />}
+        centered={true}
+      />
+      )}
```

Comment:
Couldn't all this logic be extracted to the `ScreenSplit` component? It feels a bit messy to have it here

Reply:
I agree, but Kim is doing responsive screen split, so I didn't want to step on his territory. Also screen split it shared among other Views, so any changes there shouldn't break other Views. Fiddling with that can go out of scope quite quickly.